aiopg/pool.py

```diff
@@ -187,6 +189,10 @@ def _fill_free_pool(self, override_min):
             conn = self._free[-1]
             if conn.closed:
                 self._free.pop()
+            elif self._recycle > -1 \
```

Comment:
I think this check a bit fragile, what will happened if someone pass `pool_recycle=-0.5`, I suggest change default and add parameter validation.

Reply:
I've used code logic from SQLAlchemy for compatibility - see: https://github.com/zzzeek/sqlalchemy/blob/master/lib/sqlalchemy/pool.py#L623

But I also think, that `None` by default and comparision `self._recycle is not None and self._recycle > 0` will be better.

Reply:
make sense, lets leave `-1`